src/Agent.Listener/JobDispatcher.cs

```diff
@@ -466,7 +466,19 @@ private async Task RunAsync(Pipelines.AgentJobRequestMessage message, WorkerDisp
                     // send notification to machine provisioner.
                     var systemConnection = message.Resources.Endpoints.SingleOrDefault(x => string.Equals(x.Name, WellKnownServiceEndpointNames.SystemVssConnection, StringComparison.OrdinalIgnoreCase));
                     var accessToken = systemConnection?.Authorization?.Parameters["AccessToken"];
-                    await notification.JobStarted(message.JobId, accessToken, systemConnection.Url);
+                    VariableValue buildId = null;
+                    VariableValue definitionId = null;
+                    if (message.Plan.PlanType == "Build")
```

Comment:
What is the planType when new pipelines are run? I'm guessing that it is build, but can you check that once?

Reply:
Yes it is build only verified it